src/styles/blocks/page.scss

```
@@ -0,0 +1,24 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+}
```

use `reset.scss` file for it, not `page.scss`
```suggestion
body,
h2,
p,
ul {
  margin: 0;
}
```